rest-server/README.md

```diff
@@ -175,30 +236,60 @@ Configure the rest server port in [services-configuration.yaml](../cluster-confi
 
     *Response if succeeded*
     ```
-	Status: 200
-	
+	  Status: 200
+
     {
       "message": "remove successfully"
     }
     ```
 
-    *Response if an error occured*
+	  *Response if not authorized*
     ```
-    Status: 500
+    Status: 401
 
     {
-      "error": "RemoveFailed",
-      "message": "remove failed"
+      "code": "UnauthorizedUserError",
+      "message": "Guest is not allowed to do this operation"
     }
     ```
-	
-	*Response if not authorized*
+
+	  *Response if user has no permission*
+    ```
+    Status: 403
+
+    {
+      "code": "ForbiddenUserError",
+      "message": "Non-admin is not allow to do this operation"
```

Comment:
Some messages end with `.` punctuation mark but some don't. Can you make them consistent?

Reply:
Please also change the existing messages for successful response.

Reply:
@abuccts Messages for successful response checked.